tests/src/tests/intents/swap_ft/env.rs

```diff
@@ -45,6 +45,18 @@ impl Env {
             .set_min_ttl(self.intent.id(), min_ttl)
             .await;
     }
+
+    pub async fn ft_mint(
+        &self,
+        token: &AccountId,
```

Comment:
`ft_mint` calls could be written in one line again in the tests if you change the type of the argument to `&Contract` 😄

Reply:
You're right, but in some places it's still required to be able to use with `#[rstest]`:
https://github.com/defuse-protocol/defuse-contracts/blob/72bf146a9479d85b793ebc8357343aaf3fbc3e71/tests/src/tests/intents/swap/mod.rs#L78-L88